modules/vulnerabilities/unix/nfs/nfs_overshare/templates/exports.erb

```diff
@@ -1,9 +1,12 @@
+<%require 'json' -%>
+<%$parsed_inputs = JSON.parse(@json_inputs) -%>
+<%$storage_directory = $parsed_inputs['storage_directory'].first -%>
 # /etc/exports: the access control list for filesystems which may be exported
 #               to NFS clients.  See exports(5).
 #
 # Example for NFSv2 and NFSv3:
- /exports       172.0.0.0/8(rw,sync,no_subtree_check)
- /exports       192.0.0.0/8(rw,sync,no_subtree_check)
+ <%=$storage_directory %>       172.0.0.0/8(rw,sync,no_subtree_check)
```

Comment:
I think we should be using 0.0.0.0 so we are not tying ourselves to those two network ranges